framework/web/AssetManager.php

```diff
@@ -468,7 +475,14 @@ protected function publishDirectory($src, $options)
         $dstDir = $this->basePath . DIRECTORY_SEPARATOR . $dir;
         if ($this->linkAssets) {
             if (!is_dir($dstDir)) {
-                symlink($src, $dstDir);
+                $symlink= @ symlink($src, $dstDir);
+                if (!$symlink) {
+                    if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+                        exec('mklink /J '.$dstDir.' '.$src);
```

Comment:
It is preferred to use `/D` i.e. directory symlinks instead of outdated junctions.